tests/test_ajax2.py

```diff
@@ -813,6 +813,10 @@ def test_updatelastpage(test_client, test_user_1, runestone_db_tools):
         db(
             (db.user_sub_chapter_progress.user_id == test_user_1.user_id)
             & (db.user_sub_chapter_progress.sub_chapter_id == "subchapter_a")
+            & (
```

Comment:
It would be nice to have a test case showing before and after behavior -- a test that fails before this fix, since progress is marked on two different courses which share the same base course, vs. no failure afteward.